.github/workflows/generate-builds.yml

```diff
@@ -229,48 +229,48 @@ jobs:
           soh.nro
           soh.otr
           readme.txt
-  # build-wiiu:
```

Comment:
I assume we’ll remove this block before merge?

Reply:
It's already been removed, this is the part of the diff that says it's been removed lol.

Reply:
Oh, derp.